src/components/message-display-rules/DisplayRule.tsx

```diff
@@ -0,0 +1,77 @@
+/** *****************************************************************************
+ * Copyright 2020-2020 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************** */
+
+import React, { useRef, useState } from 'react';
+import { useOutsideClickListener } from '../../hooks';
+import { MessageDisplayRule } from '../../models/EventMessage';
+import WorkspaceStore from '../../stores/workspace/WorkspaceStore';
+import NewRuleForm from './NewRuleForm';
+
+type DisplayRuleProps = {
+	rule: MessageDisplayRule;
+	activeWorkspace: WorkspaceStore;
+};
+
+const DisplayRuleRenderer = ({ rule, activeWorkspace }: DisplayRuleProps) => {
+	return (
+		<>
+			<p className='rule__session'>{rule.session}</p>
+			<p className='rule__type'>{rule.viewType}</p>
+			{rule.removable && (
+				<button
+					className='rule__delete'
+					title='delete rule'
+					onClick={() => {
+						activeWorkspace.deleteMessagesDisplayRule(rule);
+					}}>
+					&#215;
```

Comment:
we have icons for this i think and its too small and it doesnt have a cursor pointer